sp/src/game/server/hl2/npc_metropolice.cpp

```diff
@@ -549,7 +549,11 @@ void CNPC_MetroPolice::PrescheduleThink( void )
 
 	// Look at near players, always
 	m_bPlayerIsNear = false;
+#ifdef EZ
+	if ( UTIL_PlayerByIndex( 1 ) && IRelationType( UTIL_PlayerByIndex( 1 ) ) != D_HT && ((GetState() == NPC_STATE_ALERT) || (GetState() == NPC_STATE_IDLE)) )
```

Comment:
Does this need to be OR PlayerIsCriminal()? If the player is not criminal, they're still technically hostile right?

I know we don't expect to use gordon precriminal in EZ2 but I'd rather preserve that functionality just in case